opentracing-jaxrs2-itest/resteasy/src/test/java/io/opentracing/contrib/jaxrs2/itest/resteasy/RestEasyITest.java

```diff
@@ -37,11 +44,19 @@ public void testAsyncError() {
      */
     @Test
     public void testAsyncErrorTestSpanReported() {
-        Client client = ClientBuilder.newClient();
-        Response response = client.target(url("/asyncError"))
-            .request()
-            .get();
-        response.close();
+        try {
```

Comment:
Minor, but you could use a try-with-resources here:

```java
try (CloseableHttpClient build = HttpClientBuilder.create().disableAutomaticRetries().build()) {
            Client client = new ResteasyClientBuilder().httpEngine(new ApacheHttpClient4Engine(build)).build();
            Response response = client.target(url("/asyncError"))
                .request()
                .get();
            response.close();
            response.readEntity(String.class); 

} catch (Exception ex) {
// client throws an exception if async request fails 
}
```

This way, your HttpClient still closes if you get an exception anywhere in the code. You can also do the same for the `response`, but that one is less "critical".